src/test/java/ca/qaguru/oranghrmbatch24/pages/WorkShift.java

```diff
@@ -0,0 +1,99 @@
+package ca.qaguru.oranghrmbatch24.pages;
+
+import ca.qaguru.oranghrmbatch24.library.PageBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class WorkShift extends PageBase {
+
+    public Object saveNewWorkShift;
+
+    public WorkShift(WebDriver driver) {
+        super(driver);
+    }
+
+    private final String WSAddBtn = "//*[@type='button'][@class='oxd-button oxd-button--medium oxd-button--secondary']";
+    private final String WSTxtLevel = "//*[@class='oxd-input oxd-input--active']";
+    private final String WhfromText ="//*[@id=\"app\"]/div[1]/div[2]/div[2]/div/div/form/div[2]/div/div[1]/div/div[2]/div/div/input";
+    private final String WhtoText ="//*[@id=\"app\"]/div[1]/div[2]/div[2]/div/div/form/div[2]/div/div[2]/div/div[2]/div/div/input";
```

Comment:
make xpaths dynamic so that it will be short

Reply:
done